html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Form</title>
+</head>
+<body>
+  <form action="">
+    <div>
+      Email
+    </div>
+    <label for="email">
+      <input type="text">
+    </label>
+    <div>
+      Why is coding cool?
+    </div>
+    <label for="why is coding cool">
+      <select name="options">
+        <option value="select an option">Select an option</option>
+        <option value="because it's interesting">Because it's interesting</option>
+        <option value="because it's fun">Because it's fun</option>
+        <option value="because it's powerful">Because it's powerful</option>
+      </select>
+    </label>
+    <div>
+      Favorite Language (choose one)
+    </div>
+    <label for="favorite language">
+      <input type="radio" name="language" value="html" /> HTML
+      <input type="radio" name="language" value="css" /> CSS
+      <input type="radio" name="language" value="javascript" /> JavaScript
```

Comment:
The text for each option should be in a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!